.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -269,3 +273,13 @@ runs:
       k8s_version: ${{ inputs.k8s_version }}
       git_ref: ${{ inputs.git_ref }}
       private_cluster: ${{ inputs.private_cluster }}
+  - name: log retention policy
```

Comment:
Do we have log forwarding and a retention policy setup for our private cluster? If not, we should configure that so that we can access all of the logs in one place

Reply:
Added and Updated